 config.def.h | 19 ++++++++++++++++---
 1 file changed, 16 insertions(+), 3 deletions(-)

diff --git a/config.def.h b/config.def.h
index 1355ba3..2470f6d 100644
--- a/config.def.h
+++ b/config.def.h
@@ -109,7 +120,8 @@ static WebKitFindOptions findopts = WEBKIT_FIND_OPTIONS_CASE_INSENSITIVE |
  */
 static SiteSpecific styles[] = {
 	/* regexp               file in $styledir */
-	{ ".*",                 "default.css" },
+	{ ".*",                                          "default.css" },
+	{ "^https://(www\.)?startpage.com/do/search.*$", "startpage-web.css" },
 };
 
 /* certificates */
-- 
2.35.1

